docs/README.md

```diff
@@ -59,4 +59,11 @@ To preview the changes you made for this Amplify UI Docs, you can use Amplify to
 Then you need to:
 
 1. After selecting a branch, check "Connecting a monorepo? Pick a folder." and at "Enter the root directory of your app (e.g. src/myapp)", enter "docs"
-2. Set "Environment variables" from the left sidebar. Add "SITE_URL" and set it to a string value.
+2. Set "Environment variables" from the left sidebar. Add "SITE_URL", "DOCSEARCH_DOCS_APP_ID", "DOCSEARCH_DOCS_API_KEY", "DOCSEARCH_DOCS_INDEX_NAME".
+   - "SITE_URL": feel free to use any string
+   - "DOCSEARCH_DOCS_APP_ID", "DOCSEARCH_DOCS_API_KEY", "DOCSEARCH_DOCS_INDEX_NAME". If you have them from [Algolia DocSearch](https://docsearch.algolia.com/), use them. Otherwise, please use DocSearch's [testing credentials](https://docsearch.algolia.com/docs/DocSearch-v3)
```

Comment:
The Algolia DocSearch docs linked here aren't very clear on how to use the App ID and API Key. Can we clarify this part a bit more?

Reply:
What would you suggest to be be added besides in addition to L62?

Reply:
Well I don't really have a suggestion, the line "_If you have them from Algolia DocSearch_" just isn't clear to me.

Reply:
That should be fine. That means you don't "have them from Algolia DocSearch" 😁